custom_components/toyota/manifest.json

```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/DurgNomis-drol/ha_toyota",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/DurgNomis-drol/ha_toyota/issues",
-  "requirements": ["mytoyota==2.1.1", "arrow"],
```

Comment:
Maybe we shouldn't pin the versions here anymore, but simply allow all non-major updates?
So something like `"mytoyota>=2.1.1,<3.0.0"`